conda_env/specs/__init__.py

```diff
@@ -1,11 +1,17 @@
 # -*- coding: utf-8 -*-
 # Copyright (C) 2012 Anaconda, Inc
 # SPDX-License-Identifier: BSD-3-Clause
+
+# Standard library imports
+import os
+
+# Local imports
```

Comment:
The separation and import organization is good. Let's leave off the comments for the sections.